gitflow_toolbox/common/get_current_gitlab.py

```diff
@@ -10,7 +12,7 @@ def get_current_gitlab() -> gitlab.Gitlab:
         gitlab.Gitlab: current gitlab instance (or 'source', 'from' gitlab)
     """
     # If running in a gitlab CI job
-    if get_env("GITLAB_CI"):
```

Comment:
get_env check pas `os.getenv` ?

Reply:
get_env was raising an exception if variable is unset (this is the expected behaviour)
For this, L15, I just want to see if GITLAB_CI is set (true in a runner)